src/__tests__/integration/ZBWorker-integration.spec.ts

```diff
@@ -1,5 +1,4 @@
 import { ZBClient } from '../..'
-jest.unmock('node-grpc-client')
```

Comment:
Do we need to make sure the new client is mocked and then unmock it here?

Reply:
Hmm I removed the dependency to "node-grpc-client" so I removed the unmocks as well...

Reply:
If we are removing node-grpc-client, then you should remove the `src/__mocks__/node-gprc-client.ts` as well.

Reply:
ahh thanks, missed that